### DISCUSSION TITLE
Netflow V9 decoding error

### DISCUSSION THREAD

**Discussion by Tayyab146** [akvorado/akvorado#1487] [Q&A]
Hi All, I'm badly stuck with below errors and not able to figure, have used snmpv2 and snmpv3 private communities as well but no luck, can someone help with that?

[root@test-vm-10:~# curl -s http://10.20.5.60:8081/api/v0/inlet/metrics | grep akvorado_inlet' | grep _error
akvorado_inlet_core_flows_errors_total{error="SNMP cache miss", exporter="66.xx"} 185061
akvorado_inlet_flow_decoder_errors_total{name="netflow"} 320
akvorado_inlet_flow_decoder_netflow_errors_totalferror="NetFlow v9 decoding error", exporter="66.xx"} 320 akvorado_inlet_metadata_provider_snmp_poller_error_requests_total{error="get", ex
porter="66.xx"} 22631

**Comment by dbray925** [comment]
Duplicate of: https://github.com/akvorado/akvorado/discussions/1465
